source/_docs/global-cdn.md

```diff
@@ -102,7 +102,7 @@ Look for site thumbnails that show **Global CDN Upgrade Available** in your User
         192.123.456.789 example.com
         ```
 
-      4. Test your site locally by entering your domain in the browser. Once you have finished testing, remove the edits made to your hosts file.
+      4. Test HTTPS locally by entering your domain in the browser. If the site is accessible over HTTPS, the test was successful. Remove the edits made to your hosts file once you have finished testing.
```

Comment:
Maybe we can include link to https://support.google.com/chrome/answer/95617?hl=en or add similar type of visual "Secure <green lock>"